.github/scripts/context_chunker.py

```
@@ -0,0 +1,87 @@
+            "review_comments",
+            "test_failures",
+            "changed_files",
+            "ci_logs",
+            "full_diff",
+        ])
+        # Map chunk type to priority index (lower is higher priority)
+        self.priority_map: Dict[str, int] = {name: i for i, name in enumerate(self.priority_order)}
+
+        # Setup tokenizer/encoder if tiktoken available
+        self._encoder = None
+        if TIKTOKEN_AVAILABLE:
```

Ruff (0.14.6)

Undefined name `TIKTOKEN_AVAILABLE`

(F821)